### ISSUE TITLE
[Bug] Crane mixed controlls

### ISSUE THREAD

**Issue by EVGSTORE** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#206]
### Describe the bug

TLDR: [Space] keybind is used twice for brakeing and hook activation on small crane

Using the crane (small crane) has a problem currently on the keybinds. Space is used twice for the hook and the brake. whenever you do the brake the hook is activated and whenever you do the hook the brake is actvated. this is causing problems for moving the crane around while holding stuff, also for custom vehicles using the crane for the hook. 

### Steps to reproduce the behavior

1. go to any maps (ex:lees) with the small crane
2. get into the vehicle
3. look at the keybinds, the space keybind is used twice

### Expected behavior

the hook and brake keybinds are expected to have different keybinds

### Environment

latest (expermential) 0.9.6 teardown

### Additional context

suggested to add custom keybinds for vehicles or change the hook keybind on all vehicles to be something like "H" or soemthing. This is causing problems if a person is trying to lift soemthing moving then stoping, also causes problems for custom vehicles such as a winch the brake activates the winch hook. Thankyou.

**Comment by tuxedolabs** [comment]
Space is intentionally used for both handbrake and hook
